tests/components/samsungtv/test_init.py

```diff
@@ -135,7 +135,7 @@ async def _mock_async_get_discovery_info_by_st(hass: HomeAssistant, mock_st: str
         await hass.async_block_till_done()
         await hass.async_block_till_done()
 
-    assert hass.states.get("media_player.any")
+    assert hass.states.get("media_player.unnamed_device")
```

Comment:
For some strange reason, this device was registered under the name unnamed device. Does this have to do with device registry? When does a name reset?